File: API/DTOs/Statistics/ServerStatistics.cs
using System.Collections.Generic;

namespace API.DTOs.Statistics;

public class ServerStatistics
{
    public long ChapterCount { get; set; }
    public long VolumeCount { get; set; }
    public long SeriesCount { get; set; }
    public long TotalFiles { get; set; }
    public long TotalSize { get; set; }
    public long TotalGenres { get; set; }
    public long TotalTags { get; set; }
    public long TotalPeople { get; set; }

}

File: UI/Web/src/app/statistics/_models/server-statistics.ts
import {StatCount} from "./stat-count";
import {User} from "../../_models/user/user";
import {Series} from "../../_models/series";
import {Library} from "../../_models/library/library";

export interface ServerStatistics {
    chapterCount: number;
    volumeCount: number;
    seriesCount: number;
    totalFiles: number;
    totalSize: number;
    totalGenres: number;
    totalTags: number;
    totalPeople: number;
    totalReadingTime: number;
    mostActiveUsers: Array<StatCount<User>>;
    mostActiveLibraries: Array<StatCount<Library>>;
    mostReadSeries: Array<StatCount<Series>>;
    recentlyRead: Array<Series>;
}
